server/routes/routerRequestHandlers.ts

```diff
@@ -221,6 +222,29 @@ const retrieveFunctionalSkillsIfNotInSession = (curiousService: CuriousService):
   }
 }
 
+/**
```

Comment:
This file (`routerRequestHandlers.ts`) and it's test are getting massive! And not in a [good way](https://www.youtube.com/watch?v=O7TklQTeuSE)!

I'll raise a jira shortly to split these out into their own source files etc 👍

Reply:
I've raised RR-750 for this refactoring work 👍